### ISSUE TITLE
Increasing Validation Loss, Low Test Set Accuracy

### ISSUE THREAD

**Issue by jkind9** [KinWaiCheuk/Triplet-net-keras#2]
Hi,

Just a quick question about this implementation. In your code you have the Siamese net train for 500 epochs, but tracking the validation loss shows that it dramatically increases in this time. Can you elaborate if this is desirable?

Also, as you use the MNIST test data as the validation set, as far as I can tell you haven't run any unseen data through the embedding procedure, meaning its impossible to tell if a new data point would be correctly classified. I've found in my dataset (not MNIST) I have an incredibly low test set accuracy (33%). Any chance you could comment on this?

Thanks
Jkind9
